### PR TITLE
Create 10-12-2020-Mirror-For-Game.md

### PR DESCRIPTION
**Pull request by hsravat-4590**
Just anothe blog post. This one speaks about playerlink and why we had to abandon it

### CODE REVIEW

File: _posts/blog/10-12-2020-Mirror-For-Game.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Migrating to Mirror
+category: blog
+excerpt: Bye PlayerLink... For now :-(
+author: Hanzalah Ravat
+---
+
+Hi everyone. We have been working on the project for around two months now and our progress is comming along smoothly. However, As we are approaching many deadlines we have had to make some decisions regarding PlayerLink and our game. This blog post aims to address these decisions
```
Comment (nisathnasar) [inline _posts/blog/10-12-2020-Mirror-For-Game.md:9]: coming*

File: _posts/blog/10-12-2020-Mirror-For-Game.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Why Migrate to Mirror
+Unfortunately, we have strict deadlines to meet with the Cops And Robbers project which dictates the majority of decisions we make with the project. In this case, we have to deliver an MVP next week and we must have some form of multiplayer implemented by then and with PlayerLink at such a primitive stage with limited test data available, we have had to look elsewhere
+
+### Enter Mirror
+After a lot of consideration, we have decided to use Mirror to implement Multiplayer in our game. We decided to use Mirror as it is based on Unet which provides us with access to comprehensive documentation and is Open Source allowing us to ensure that the code is sound and secure. Mirror also uses a single codebase for both the client and server reducing the number of projects to maintain
```
Comment (nisathnasar) [inline _posts/blog/10-12-2020-Mirror-For-Game.md:18]: UNet*

File: _posts/blog/10-12-2020-Mirror-For-Game.md (hunk 1 above)
Comment (hsravat-4590) [reply _posts/blog/10-12-2020-Mirror-For-Game.md:9]: I'll fix that now

File: _posts/blog/10-12-2020-Mirror-For-Game.md (hunk 2 above)
Comment (hsravat-4590) [reply _posts/blog/10-12-2020-Mirror-For-Game.md:18]: I'll fix that now

**Review by ellimanh — APPROVED** [review]
LGTM!

**Review by Mirlle — APPROVED** [review]

**Pull request merged**